examples/sst2/train_test.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for sst2.train."""
+from absl.testing import absltest
+from absl.testing import parameterized
+from configs import default as default_config
```

Comment:
nit: I really like ordered imports like this:

```
from absl.testing import absltest
from absl.testing import parameterized
import jax
import jax.test_util
import numpy as np

from configs import default as default_config
import train
```

Reply:
I agree with you, but the linter doesn't agree with us :) I think i'd like to stick with keeping the linter happy.